src/screens/StudyScreen/index.jsx

```diff
@@ -317,11 +317,11 @@ const StudyScreen = ({ navigation, route }) => {
       }
     };
 
-  const onCopyText = text => () => {
+  const onCopyText = (text) => {
```

Comment:
If the function is declared in this signature, then the component will be: `<Text callback={() => onCopyText(myWordData)} />` instead of `<Text callback={onCopyText(myWordData)} />`.
As the number of props increases, this can hurt readability.